components/organisms/bigCalendar/README.md

```diff
@@ -0,0 +1,5 @@
+# Accordion
+
+## Links
+
+- [Bootstrap Accordion](https://getbootstrap.com/docs/4.3/components/collapse/#accordion-example)
```

Comment:
This was a mistake I had to change since I copied from accordion to have the "scheleton" and I didnt change the readme information with the stories I wanted to create